### ISSUE TITLE
DOMException: Failed to execute 'appendChild' on 'Node': The new child element contains the parent.

### ISSUE THREAD

**Issue by phorward** [viur-framework/viur-vi#88]
I'm recognizing this error when this numericBone is active:
![shot-2020-10-16_16-45-56](https://user-images.githubusercontent.com/16870072/96273115-45e1b980-0fcf-11eb-939b-792d06fef9c5.jpg)

The traceback is very incomprehensible, and does not help much.
```pyodide.asm.js:8 Uncaught DOMException: Failed to execute 'appendChild' on 'Node': The new child element contains the parent.
    at _hiwire_call_member (http://localhost:8080/pyodide/pyodide.asm.js:8:31195)
    at _JsBoundMethod_Call (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[325]:0xfb707)
    at http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[14894]:0x973c2e
    at __PyObject_FastCallKeywords (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[841]:0x133ca7)
    at _call_function (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[2755]:0x327e2b)
    at __PyEval_EvalFrameDefault (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[2750]:0x3206a0)
    at http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[20581]:0x98721c
    at __PyEval_EvalCodeWithName (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[2745]:0x314c59)
    at __PyFunction_FastCallKeywords (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[842]:0x133f3d)
    at _call_function (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[2755]:0x327e1f)
_hiwire_call_member @ pyodide.asm.js:8
_JsBoundMethod_Call @ pyodide.asm.wasm:0xfb707
(anonymous) @ pyodide.asm.wasm:0x973c2e
__PyObject_FastCallKeywords @ pyodide.asm.wasm:0x133ca7
_call_function @ pyodide.asm.wasm:0x327e2b
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x3206a0
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133f3d
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x32068f
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallDict @ pyodide.asm.wasm:0x132898
__PyObject_FastCallDict @ pyodide.asm.wasm:0x1322de
__PyObject_Call_Prepend @ pyodide.asm.wasm:0x1353f6
_method_call @ pyodide.asm.wasm:0x136d3a
(anonymous) @ pyodide.asm.wasm:0x9743f5
_PyObject_Call @ pyodide.asm.wasm:0x134af1
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x320d48
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallDict @ pyodide.asm.wasm:0x132898
__PyObject_FastCallDict @ pyodide.asm.wasm:0x1322de
__PyObject_Call_Prepend @ pyodide.asm.wasm:0x1353f6
_slot_tp_init @ pyodide.asm.wasm:0x1d186a
(anonymous) @ pyodide.asm.wasm:0x97655c
_type_call @ pyodide.asm.wasm:0x1e07d6
(anonymous) @ pyodide.asm.wasm:0x9762ee
__PyObject_FastCallKeywords @ pyodide.asm.wasm:0x133ca7
_call_function @ pyodide.asm.wasm:0x327e2b
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x3206fe
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133f3d
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x32068f
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133f3d
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x320764
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133f3d
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x320764
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133f3d
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x320764
(anonymous) @ pyodide.asm.wasm:0x98721c
_function_code_fastcall @ pyodide.asm.wasm:0x13359d
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133eb2
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x3206fe
(anonymous) @ pyodide.asm.wasm:0x98721c
_function_code_fastcall @ pyodide.asm.wasm:0x13359d
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133eb2
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x32068f
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallDict @ pyodide.asm.wasm:0x132898
__PyObject_FastCallDict @ pyodide.asm.wasm:0x1322de
__PyObject_Call_Prepend @ pyodide.asm.wasm:0x1353f6
_method_call @ pyodide.asm.wasm:0x136d3a
(anonymous) @ pyodide.asm.wasm:0x9743f5
_PyObject_Call @ pyodide.asm.wasm:0x134af1
__pyproxy_apply @ pyodide.asm.wasm:0xfc26d
Module.__pyproxy_apply @ pyodide.asm.js:8
apply @ pyodide.asm.js:8
XMLHttpRequest.send (async)
_hiwire_call_member @ pyodide.asm.js:8
_JsBoundMethod_Call @ pyodide.asm.wasm:0xfb707
(anonymous) @ pyodide.asm.wasm:0x973c2e
__PyObject_FastCallKeywords @ pyodide.asm.wasm:0x133ca7
_call_function @ pyodide.asm.wasm:0x327e2b
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x3206a0
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallDict @ pyodide.asm.wasm:0x132898
__PyObject_FastCallDict @ pyodide.asm.wasm:0x1322de
__PyObject_Call_Prepend @ pyodide.asm.wasm:0x1353f6
_method_call @ pyodide.asm.wasm:0x136d3a
(anonymous) @ pyodide.asm.wasm:0x9743f5
_PyObject_Call @ pyodide.asm.wasm:0x134af1
__pyproxy_apply @ pyodide.asm.wasm:0xfc26d
Module.__pyproxy_apply @ pyodide.asm.js:8
apply @ pyodide.asm.js:8
_hiwire_call_member @ pyodide.asm.js:8
_JsBoundMethod_Call @ pyodide.asm.wasm:0xfb707
(anonymous) @ pyodide.asm.wasm:0x973c2e
__PyObject_FastCallKeywords @ pyodide.asm.wasm:0x133ca7
_call_function @ pyodide.asm.wasm:0x327e2b
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x3206a0
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133f3d
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x320764
(anonymous) @ pyodide.asm.wasm:0x98721c
_function_code_fastcall @ pyodide.asm.wasm:0x13359d
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133eb2
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x32068f
(anonymous) @ pyodide.asm.wasm:0x98721c
_function_code_fastcall @ pyodide.asm.wasm:0x13359d
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133eb2
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x32068f
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallDict @ pyodide.asm.wasm:0x132898
__PyObject_FastCallDict @ pyodide.asm.wasm:0x1322de
__PyObject_Call_Prepend @ pyodide.asm.wasm:0x1353f6
_slot_tp_init @ pyodide.asm.wasm:0x1d186a
(anonymous) @ pyodide.asm.wasm:0x97655c
_type_call @ pyodide.asm.wasm:0x1e07d6
(anonymous) @ pyodide.asm.wasm:0x9762ee
__PyObject_FastCallKeywords @ pyodide.asm.wasm:0x133ca7
_call_function @ pyodide.asm.wasm:0x327e2b
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x3206fe
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133f3d
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x320764
(anonymous) @ pyodide.asm.wasm:0x98721c
_function_code_fastcall @ pyodide.asm.wasm:0x13359d
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133eb2
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x32068f
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallDict @ pyodide.asm.wasm:0x132898
__PyObject_FastCallDict @ pyodide.asm.wasm:0x1322de
__PyObject_Call_Prepend @ pyodide.asm.wasm:0x1353f6
_slot_tp_init @ pyodide.asm.wasm:0x1d186a
(anonymous) @ pyodide.asm.wasm:0x97655c
_type_call @ pyodide.asm.wasm:0x1e07d6
(anonymous) @ pyodide.asm.wasm:0x9762ee
_PyObject_Call @ pyodide.asm.wasm:0x134af1
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x320d48
(anonymous) @ pyodide.asm.wasm:0x98721c
_function_code_fastcall @ pyodide.asm.wasm:0x13359d
__PyFunction_FastCallKeywords @ pyodide.asm.wasm:0x133eb2
_call_function @ pyodide.asm.wasm:0x327e1f
__PyEval_EvalFrameDefault @ pyodide.asm.wasm:0x32068f
(anonymous) @ pyodide.asm.wasm:0x98721c
__PyEval_EvalCodeWithName @ pyodide.asm.wasm:0x314c59
__PyFunction_FastCallDict @ pyodide.asm.wasm:0x132898
__PyObject_FastCallDict @ pyodide.asm.wasm:0x1322de
__PyObject_Call_Prepend @ pyodide.asm.wasm:0x1353f6
_method_call @ pyodide.asm.wasm:0x136d3a
(anonymous) @ pyodide.asm.wasm:0x9743f5
_PyObject_Call @ pyodide.asm.wasm:0x134af1
__pyproxy_apply @ pyodide.asm.wasm:0xfc26d
Module.__pyproxy_apply @ pyodide.asm.js:8
apply @ pyodide.asm.js:8
Show 130 more frames
```

**Comment by phorward** [comment]
Error recognized again in a different project, traceback
```
Uncaught DOMException: Failed to execute 'appendChild' on 'Node': The new child element contains the parent.
    at _hiwire_call_member (http://localhost:8080/pyodide/pyodide.asm.js:8:31195)
    at _JsBoundMethod_Call (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[325]:0xfb707)
    at http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[14894]:0x973c2e
    at __PyObject_FastCallKeywords (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[841]:0x133ca7)
    at _call_function (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[2755]:0x327e2b)
    at __PyEval_EvalFrameDefault (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[2750]:0x3206a0)
    at http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[20581]:0x98721c
    at __PyEval_EvalCodeWithName (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[2745]:0x314c59)
    at __PyFunction_FastCallKeywords (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[842]:0x133f3d)
    at _call_function (http://localhost:8080/pyodide/pyodide.asm.wasm:wasm-function[2755]:0x327e1f)
```

**Comment by Grashalmbeisser** [comment]
Hacky Workaround: 

You can change the "required" variable to False and the error will not occure again. 
idk. why it is working. 
Tested in an Incognito Tab. 

(Maybe you could head off wrong inputs in the toDB Method?)

**Comment by xnopasaranx** [comment]
This seems to be connected to the List handler. I can get rid of the error by removing the according numericBone from adminInfo columns and everything works. It seems to be a display error. filter orderby works eventhough the column is not shown.
(in the project I am debugging, the bone this error concerns is used as sortpoints btw.)

**Comment by phorward** [comment]
Bug fixed by b11c0d9224c2eedd67fa9bbdb53672f70aaaeaa2
